CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- BREAKING: Remove legacy node selector on `RoleGroup` ([#652]).
```

Comment:
Is it "really" breaking if we remove deprecated stuff? If yes can you add the crd-change/breaking label?
```suggestion
- BREAKING: Remove legacy node selector on `RoleGroup` ([#652]).
```

Reply:
I would consider it breaking - for users as well as our operators as it's going to break stuff. I think the deprecation is the call out that there will be a breaking change in the future.
Added the labels

Reply:
Agreed with @sbernauer. _Especially_ since we don't currently do a great job warning users that they're using deprecated features.